website/source/docs/providers/cloudstack/r/affinity_group.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "cloudstack"
+page_title: "CloudStack: cloudstack_affinity_group"
+sidebar_current: "docs-cloudstack-resource-affinity-group"
+description: |-
+  Creates an affinity group.
+---
+
+# cloudstack\_affinity\_group
+
+Creates an affinity group.
+
+## Example Usage
+
+```
+resource "cloudstack_affinity_group" "default" {
+  name = "myGroup"
+  type = "host anti-affinity"
```

Comment:
Is this a valid type? I thought only `affinity` and `anti-affinity` were valid types?